galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.jmeter.manager/src/main/java/dev/galasa/jmeter/IJMeterSession.java

```diff
@@ -17,68 +20,81 @@
      * This method gets called before start of a session providing necessary properties
      * to the JMeter session
      * @param properties A map of Strings with provided properties
+     * @throws JMeterManagerException
      */
-    public void applyProperties(Map<String, String> properties);
+    public void applyProperties(InputStream propStream) throws JMeterManagerException;
 
     /**
      * Start up a jmeter thread to run through the lifetime of the tests
      * Usage of cps.properties with their own default location for JMX files...
+     * @throws JMeterManagerException
      */
-    public void startJmeter();
+    public void startJmeter(int timeout) throws JMeterManagerException;
 
     /**
      * Giving jmeter a deadline before it needs to be finished and the test gets shutdown
+     * @throws JMeterManagerException
```

Comment:
What is the deadline if there is no timeout

Reply:
Added a default version with 60s for starting jmeter and one with paramater, removed waitingForJmeter as this gets done in the startJMeter()